synapse/replication/http/__init__.py

```diff
@@ -34,9 +34,12 @@ def __init__(self, hs):
 
     def register_servlets(self, hs):
         send_event.register_servlets(hs, self)
-        membership.register_servlets(hs, self)
         federation.register_servlets(hs, self)
-        login.register_servlets(hs, self)
-        register.register_servlets(hs, self)
-        devices.register_servlets(hs, self)
-        streams.register_servlets(hs, self)
+
+        # The following can't currently be instansiated on master.
+        if hs.config.worker.worker_app is None:
```

Comment:
But if `hs.config.worker.worker_app` is None, aren't we running on the master?

Reply:
That's mean to read "The following can't currently be instantiatedc on **workers**

Reply:
Fixed?